dev-libs/libical/libical-2.0.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="Eric Busboom"
+LICENSE="MPL v1.0
+	GNU LGPL v2.1"
+REVISION="1"
+scrGitRev="6c4af23b0a95fd105f38e879908cbc80390f3716"
+SOURCE_URI="https://github.com/libical/libical/releases/download/v$portVersion/libical-$portVersion.tar.gz"
+CHECKSUM_SHA256="654c11f759c19237be39f6ad401d917e5a05f36f1736385ed958e60cf21456da"
+SOURCE_DIR="libical-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	libical$secondaryArchSuffix = $portVersion
+	lib:libical$secondaryArchSuffix
```

Comment:
Check if the library has a soname (there is a file called libical.so.1.2.3 and libical.so.1, for example) and add the matching "= 1.2.3 compat >= 1" to the libraries.